Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+M1AeonAttackWarning = {
+    {text = '[Benson]: Aeon units are closing it to your position, get ready to fight.', vid = 'N03_M1AeonAttackWarning.sfd', bank = 'N03_VO', cue = 'M1AeonAttackWarning', faction = 'UEF'},
+}
+
+
+
+-- Player sees the dead UEF Base
+M1UEFBaseDialogue = {
+    {text = '[Anyone]: Something about the dead base.', vid = 'N03_M1UEFBaseDialogue.sfd', bank = 'N03_VO', cue = 'M1UEFBaseDialogue', faction = 'UEF'},
+}
+
+
+
+-- Once player scouts the civilian city, add objective to capture one building
+M1SecondaryObjective = {
+    {text = '[Benson]: We\'ve located a small civilian settlement south of your position. Capture the marked building to get an additional intel on this area.', vid = 'N03_M1SecondaryObjective.sfd', bank = 'N03_VO', cue = 'M1SecondaryObjective', faction = 'UEF'},
+}
+
+-- When player captured the building
+M1SecondaryDone = {
+    {text = '[Benson]: Building captured...', vid = 'N03_M1SecondaryDone.sfd', bank = 'N03_VO', cue = 'M1SecondaryDone', faction = 'UEF'},
+}
+
+-- When the civ building dies
+M1SecondaryFailed = {
+    {text = '[Benson]: The building is destoryed. No intel for us.', vid = 'N03_M1SecondaryFailed.sfd', bank = 'N03_VO', cue = 'M1SecondaryFailed', faction = 'UEF'},
+}
+
+-- Remind to capture the building
+M1SecondaryReminder = {
+    {text = '[Benson]: Capture the Aeon building to gather extra intel.', vid = 'N03_M1SecondaryReminder.sfd', bank = 'N03_VO', cue = 'M1SecondaryReminder', faction = 'UEF'},
+}
+
+
+
+-- Unlock T2 static shield
+M1ShieldUnlock = {
+    {text = '[Benson]: T2 Shield uploaded.', vid = 'N03_M1ShieldUnlock.sfd', bank = 'N03_VO', cue = 'M1ShieldUnlock', faction = 'UEF'},
+}
+
+-- Bonus ojd done, construct shield over the ship
+M1ShieldConstructed = {
+    {text = '[Benson]: The shield should make sure that the ship survives longer.', vid = 'N03_M1ShieldConstructed.sfd', bank = 'N03_VO', cue = 'M1ShieldConstructed', faction = 'UEF'},
+}
+
+
+
+-- Tell the player we're sending engineers
+M1Enginners1 = {
+    {text = '[Benson]: Commander, we\'re dispatching a group of engineers to your location to investigate the ship\'s damage. They\'ll arive shortly.', vid = 'N03_M1Enginners1.sfd', bank = 'N03_VO', cue = 'M1Enginners1', faction = 'UEF'},
+}
+
+-- When the transports appear at the map
+M1Enginners2 = {
+    {text = '[Benson]: The transports are inbound.', vid = 'N03_M1Enginners2.sfd', bank = 'N03_VO', cue = 'M1Enginners2', faction = 'UEF'},
+}
+
+-- Engineers start repairing the ship
+M1Enginners3 = {
+    {text = '[Benson]: Scanning the ship for damage.', vid = 'N03_M1Enginners3.sfd', bank = 'N03_VO', cue = 'M1Enginners3', faction = 'UEF'},
+}
+
+-- Few seconds after, assign objective to reclaim crystal
+M1CrystalsObjective = {
+    {text = '[Benson]: Ship is very damaged, to repair it we must reclaim some crystals, etc etc...', vid = 'N03_M1CrystalsObjective.sfd', bank = 'N03_VO', cue = 'M1CrystalsObjective', faction = 'UEF'},
```

Comment:
or "The scan of the ship is completed, it doesnt look good commander. [...]"
As the fact that its heavily damaged is shown quite well by the HP